content/using/os/using-your-ship.md

```diff
@@ -0,0 +1,327 @@
++++
+title = "Using Your Ship"
```

Comment:
Perhaps a better title for this would be "Basics" or "Getting Started"? As in, this is the logical starting point when wanting to dig in to learning more about using your ship. Thoughts?